package.json

```diff
@@ -22,72 +22,79 @@
-    "babel-core": "^6.21.0",
-    "babel-helper-builder-react-jsx": "^6.22.0",
-    "babel-helpers": "^6.22.0",
-    "babel-jest": "^18.0.0",
-    "babel-loader": "^6.2.10",
-    "babel-plugin-transform-object-rest-spread": "^6.20.2",
-    "babel-plugin-transform-react-display-name": "^6.22.0",
-    "babel-plugin-transform-react-jsx": "^6.22.0",
-    "babel-preset-es2015": "^6.18.0",
-    "babel-preset-react": "^6.16.0",
-    "babel-register": "^6.22.0",
-    "css-loader": "^0.26.1",
+    "@babel/core": "^7.5.5",
+    "@babel/helper-builder-react-jsx": "^7.0.0",
+    "@babel/helpers": "^7.0.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.5.0",
+    "@babel/plugin-transform-react-display-name": "^7.0.0",
+    "@babel/plugin-transform-react-jsx": "^7.0.0",
+    "@babel/polyfill": "^7.4.4",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/register": "^7.5.5",
+    "@types/optimize-css-assets-webpack-plugin": "^5.0.0",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^24.9.0",
+    "babel-loader": "^8.0.6",
+    "css-loader": "^3.2.0",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
-    "eslint": "^4.19.1",
+    "eslint": "^6.3.0",
     "eslint-config-airbnb": "^16.1.0",
-    "eslint-plugin-import": "^2.10.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.7.0",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.14.3",
     "espree": "^3.3.2",
-    "extract-text-webpack-plugin": "^1.0.1",
+    "fancy-log": "^1.3.3",
     "file-loader": "^0.9.0",
-    "gulp": "^3.9.1",
+    "gulp": "^4.0.2",
+    "gulp-cli": "^2.2.0",
     "gulp-util": "^3.0.7",
-    "jest": "^20.0.0",
+    "jest": "^24.9.0",
     "jest-enzyme": "^2.1.2",
     "jest-junit": "^6.4.0",
     "jsdom": "^8.4.0",
     "json-loader": "^0.5.4",
-    "node-sass": "^4.11.0",
+    "mini-css-extract-plugin": "^0.8.0",
+    "node-sass": "^4.12.0",
+    "optimize-css-assets-webpack-plugin": "^5.0.3",
     "redux-mock-store": "^1.5.1",
-    "sass-loader": "^4.1.1",
-    "string-replace-loader": "^1.0.5",
-    "style-loader": "^0.13.1",
-    "url-loader": "^0.5.7",
-    "webpack": "^1.14.0"
+    "sass-loader": "^8.0.0",
+    "string-replace-loader": "^2.2.0",
+    "style-loader": "^1.0.0",
+    "url-loader": "^2.1.0",
+    "webpack": "^4.39.3"
   },
   "dependencies": {
-    "auth0-lock": "^11.8.0",
-    "babel-polyfill": "^6.20.0",
-    "brace": "^0.3.0",
-    "chart.js": "^2.5.0",
+    "auth0-lock": "^11.17.2",
+    "brace": "^0.11.1",
+    "chart.js": "^2.8.0",
     "ckeditor": "file:node_shims/ckeditor",
     "create-react-class": "^15.5.2",
     "d3": "^3.4.6",
-    "dagre-d3": "git+https://github.com/ENCODE-DCC/dagre-d3.git#v1.2.0",
-    "dalliance": "git+https://github.com/ENCODE-DCC/dalliance.git#v1.0",
+    "dagre-d3": "git+https://github.com/ENCODE-DCC/dagre-d3.git",
```

Comment:
I'm not sure if this would be better or if it matters very much, but you could make a release for your changes to `dagre-d3` and include the release number here.

Reply:
I can, but the style seems to have changed to move the changes to master, to make a release number be redundant. Should I still add it?

Reply:
Yeah, I agree with @emro on this; I know I mentioned it was probably OK to use master, but I’m not feeling that comfortable with that. I think tagging dagre-d3 with a release would be better in case we want to update master, but not cause new builds to change at that moment.

Reply:
Done. I created WRAN-1363 and WRAN-1364 with a comment in it signifying they are ENCODE-specific.